packages/verification/src/rest/v1/callbacks/callbacks-webhook.ts

```diff
@@ -1,18 +1,59 @@
 import { VerificationRequestEvent, VerificationResultEvent } from '../../../models';
+import { SinchClientParameters, validateAuthenticationHeader } from '@sinch/sdk-client';
+import { IncomingHttpHeaders } from 'http';
 
 export type VerificationCallback = VerificationRequestEvent | VerificationResultEvent;
 
-export const parseVerificationEventNotification = (eventBody: any): VerificationCallback => {
-  if (eventBody.event) {
-    switch (eventBody.event) {
-    case 'VerificationRequestEvent':
-      return eventBody as VerificationRequestEvent;
-    case 'VerificationResultEvent':
-      return eventBody as VerificationResultEvent;
-    default:
-      throw new Error(`Unknown Verification event type: ${eventBody.event}`);
+export class CallbackWebhooks {
+
+  private sinchClientParameters: SinchClientParameters;
+
+  constructor(sinchClientParameters: SinchClientParameters) {
+    this.sinchClientParameters = sinchClientParameters;
+  }
+
+  /**
+   * Validate authorization header for callback request
+   * @param {IncomingHttpHeaders} headers - Incoming request's headers
+   * @param {string} path - Incoming request's path
+   * @param {any} body - Incoming request's body
+   * @param {string} method - Incoming request's HTTP method
+   * @return {boolean} - true if the authorization header is valid
+   */
+  public validateAuthorizationHeader(
```

Comment:
`validateAuthenticationHeader` as common public interface ?

Reply:
Will be refactored in the PR#22